aea/configurations/base.py

```diff
@@ -275,7 +277,7 @@ def __init__(self, name: str, author: str, version: str):
         self.version = version
 
     @property
-    def public_id(self) -> PublicId:
+    def public_id(self) -> PublicId:  # pragma: no cover
```

Comment:
It is good to test the properties. It is clear that it returns the correct object but we could cover it and it is not trivial.